1 Quick Sort/QuickSort1.py

```diff
@@ -0,0 +1,89 @@
+import random
+import time
+import os
+numbersArr = []
+path = input("Inter path to the file:")
+
+def readArrayFromFile(numbersArr,path):
+        file = open(path +".py")
```

Comment:
Do not concatenate extension here.
If user specified wrong path (without extension) - just print the error "File wasn't found"